docker-compose.yml

```diff
@@ -130,3 +130,4 @@ services:
       SPRING_SLEUTH_ENABLED: "true"
       SPRING_SLEUTH_SAMPLER_PROBABILITY: 1
       SPRING_ZIPKIN_BASE_URL: http://zipkin:9411/
+      HOST_WORKING_DIR: "${PWD}"
```

Comment:
Why set `HOST_WORKING_DIR`?

Reply:
The paths have to be absolute paths. And the absolute path also has to work when the containers are started from inside the "java-development" container. So, had to pass the absolute path to the "java-development" container.

Reply:
Volume Paths can be relative, e.g. https://docs.docker.com/compose/compose-file/#volume-configuration-reference

```
# Path on the host, relative to the Compose file
  - ./cache:/tmp/cache
```

But you are correct on the requirement to pass `$PWD` to the `java-development` container.